src/utils/resources/vm/utils/selectors.ts

```diff
@@ -145,6 +146,11 @@ export const getWorkload = (vm: V1VirtualMachine): string =>
 export const getAccessCredentials = (vm: V1VirtualMachine): V1AccessCredential[] =>
   vm?.spec?.template?.spec?.accessCredentials;
 
+export const getIsDynamicSSHInjectionEnabled = (vm: V1VirtualMachine): boolean =>
+  vm?.metadata?.labels?.[DYNAMIC_CREDENTIALS_SUPPORT] === 'true' &&
+  !!vm?.spec?.template?.spec?.accessCredentials?.[0]?.sshPublicKey?.propagationMethod
+    ?.qemuGuestAgent;
```

Comment:
you can get this a bit more readable like this:
```
export const getIsDynamicSSHInjectionEnabled = (vm: V1VirtualMachine): boolean =>
  getLabel(vm, DYNAMIC_CREDENTIALS_SUPPORT) === 'true' && Boolean(getAccessCredentials(vm)?.[0]?.sshPublicKey?.propagationMethod?.qemuGuestAgent)
```

Reply:
thank you!